CHANGELOG.md

```diff
@@ -10,3 +10,4 @@
 * [FEATURE] You can specify "heap ballast" to reduce Go GC Churn #1489
 * [BUGFIX] HA Tracker no longer always makes a request to Consul/Etcd when a request is not from the active replica #1516
 * [BUGFIX] Queries are now correctly cancelled by the query-frontend #1508
+* [CHANGE] Added check for retention period to be a multiple of periodic table duration
```

Comment:
This should be under `master / unreleased` and not under `0.1.0`.

Reply:
Sorry, my bad. Fixed it.